tests/phpunit/tests/rest-api/wpApplicationPasswords/createNewApplicationPassword.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * Unit tests covering WP_Application_Passwords::create_new_application_password functionality.
+ *
+ * @package    WordPress
+ * @subpackage REST API
+ */
+
+/**
+ * @covers WP_Application_Passwords::create_new_application_password
+ *
+ * @group  restapi
+ * @group  app_password
+ */
+class Test_WPApplicationPasswords_CreateNewApplicationPassword extends WP_UnitTestCase {
+
+	/**
+	 * Administrator user id.
+	 *
+	 * @var int
+	 */
+	private static $user_id;
+
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		self::$user_id = $factory->user->create(
+			array(
+				'role' => 'administrator',
+			)
+		);
+
+		if ( is_multisite() ) {
+			grant_super_admin( self::$user_id );
+		}
+	}
+
+	public static function wpTearDownAfterClass() {
+		self::delete_user( self::$user_id );
+	}
```

Comment:
```suggestion
```

We do this all over the place, but it turns out to not actually be necessary. It'll be dropped by a call to `_delete_all_data` in the base teardown method. At some point we'll clean up the existing usages in the test suite.